### PR TITLE
Chat - Cut off user name on nav bar

### PR DESCRIPTION
**Pull request by davidchan1219**
**What this PR does** 📖

**Which issue(s) this PR fixes** 🔨

Fixes #

**Special notes for reviewers** 🗒️

**Additional comments** 🎤

### CODE REVIEW

**Comment by RetroPronghorn** [comment]
What are the changes to electron and lang? Should these be omitted from the PR?

**Review by stavares843 — APPROVED** [review]
Captura de ecrã 2021-12-08, às 15 01 10


before vs after

fixed ✅

**Comment by WanderingHogan** [comment]
@RetroPronghorn the submodule issue is something do with our submodules not being set up right 
https://stackoverflow.com/questions/18770545/why-is-my-git-submodule-head-detached-from-master

**Review by WanderingHogan — APPROVED** [review]

File: components/tailored/core/statusbar/Statusbar.less [hunk 1]
```diff
@@ -73,6 +73,7 @@
       text-overflow: ellipsis;
       overflow: hidden;
       white-space: nowrap;
+      line-height: 2 !important;
```
[the remark this answers is not in the document]
Comment (WanderingHogan) [reply components/tailored/core/statusbar/Statusbar.less:76]: looks like it works fine without the important property

[the remark this answers is not in the document]
Comment (stavares843) [reply components/tailored/core/statusbar/Statusbar.less:76]: cc @nickjjang

**Pull request merged**